openupgrade_scripts/scripts/base/16.0.1.3/pre-migration.py

```diff
@@ -12,6 +12,22 @@
 _logger = logging.getLogger(__name__)
 
 
+def enable_coupon_sharing_within_entity(cr):
+    """The coupon_commercial_partner_applicability module is merged into
+    loyalty_partner_applicability. It is checked if it was installed in v15
+    to set the configuration parameter to True to keep the same functionality."""
+    # Check if the module is installed and its status is "installed".
+    if openupgrade.is_module_installed(cr, "coupon_commercial_partner_applicability"):
```

Comment:
No valid in this case. Check this discussion: https://github.com/OCA/OpenUpgrade/pull/4191#discussion_r1371547207

Reply:
In that case the script was being made in the module itself, there the module is already merged, I have added the script in base, before the merge method is executed. 
https://github.com/OCA/OpenUpgrade/blob/bd4f489b6b976c703f26de8a509be362d238bb25/openupgrade_scripts/scripts/base/16.0.1.3/pre-migration.py#L133
In this case there would be two options, do it as I have done including it in apriori or do it in the module itself by calling the update_module_names method from the script but in this case it would not be included in apriori and it is clearer to always have it in apriori.

Reply:
Thanks for the explanations :)